app/src/main/java/jp/co/cyberagent/stf/Service.java

```diff
@@ -367,4 +373,69 @@ public void run() {
             }
         }
     }
+
+    /**
+     * Monitors the adb state by checking /sys/class/android_usb/android0/state
+     *
+     * If state is DISCONNECTED then IdentityActivity will be shown to allow easier identification
+     * of malfunctioning devices
+     */
+    private class AdbMonitor extends Thread {
+        // If something goes wrong you have 30 seconds to kill the STFService
```

Comment:
I'm not sure if this comment is up to date anymore now that the value has changed.

Reply:
I'm just testing it with 5s, will revert back to 30s once all is done